<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BoggleWindow</class>
 <widget class="QDialog" name="BoggleWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1128</width>
    <height>922</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="startButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>240</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>start</string>
   </property>
  </widget>
  <widget class="QLabel" name="timerLabel">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>0</y>
     <width>441</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>                                   03:00</string>
   </property>
  </widget>
  <widget class="QTimeEdit" name="timeEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>300</y>
     <width>118</width>
     <height>26</height>
    </rect>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>0</hour>
     <minute>3</minute>
     <second>0</second>
     <year>2000</year>
     <month>1</month>
     <day>1</day>
    </datetime>
   </property>
   <property name="maximumDateTime">
    <datetime>
     <hour>0</hour>
     <minute>3</minute>
     <second>0</second>
     <year>2000</year>
     <month>1</month>
     <day>1</day>
    </datetime>
   </property>
   <property name="maximumTime">
    <time>
     <hour>0</hour>
     <minute>3</minute>
     <second>0</second>
    </time>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
